add-on/chrome/bootstrap.js

```diff
@@ -533,45 +533,44 @@ var WindowListener = {
         this._hideBrowserSharingInfoBar();
 
         let box = gBrowser.getNotificationBox();
-        let pauseButtonLabel = this._getString(this._browserSharePaused ?
-                                               "infobar_button_resume_label" :
-                                               "infobar_button_pause_label");
-        let pauseButtonAccessKey = this._getString(this._browserSharePaused ?
-                                                   "infobar_button_resume_accesskey" :
-                                                   "infobar_button_pause_accesskey");
-        let barLabel = this._getString(this._browserSharePaused ?
-                                       "infobar_screenshare_paused_browser_message" :
-                                       "infobar_screenshare_browser_message2");
+        // Pre-load strings
+        let pausedStrings = {
+          label: this._getString("infobar_button_restart_label2"),
+          accesskey: this._getString("infobar_button_restart_accesskey"),
+          message: this._getString("infobar_screenshare_stop_sharing_message")
+        };
+
+        let unpausedStrings = {
+          label: this._getString("infobar_button_stop_label2"),
+          accesskey: this._getString("infobar_button_stop_accesskey"),
+          message: this._getString("infobar_screenshare_browser_message2")
+        };
+        
         let bar = box.appendNotification(
-          barLabel,
+          pausedStrings.message,
           kBrowserSharingNotificationId,
           // Icon is defined in browser theme CSS.
           null,
           box.PRIORITY_WARNING_LOW,
           [{
-            label: pauseButtonLabel,
-            accessKey: pauseButtonAccessKey,
+            label: pausedStrings.label,
+            accessKey: pausedStrings.accessKey,
```

Comment:
I believe the infobar gets reshown when you switch tabs even when you're paused. Double check that pausing then switching tabs keeps the text showing the paused state.